docs/submitting_data.rst

```diff
@@ -301,7 +307,8 @@ artifacts data structures to send, do something like this:
 
     # Send the collection to treeherder
     client = TreeherderClient(protocol='https', host='treeherder.mozilla.org')
-    client.post_collection('mozilla-central', 'oauth_key', 'oauth_secret', tac)
+    auth = TreeherderAuth('oauth_key', 'oauth_secret', 'mozilla-central')
+    client.post_collection('mozilla-central',tac, auth=auth)
```

Comment:
nit: spacing around tac. also same comment as above about passing auth when client class is created